draft-ietf-quic-tls.md

```diff
@@ -84,7 +85,7 @@ informative:
       - ins: R. Ng
       - ins: B. Tackmann
     date: 2019-06-01
-    target: https://eprint.iacr.org/2019/624
+    target: "http://dx.doi.org/10.1007/978-3-030-26948-7_9"
```

Comment:
With a DOI, you can just cite this as: `{{?NAN=DOI.10.1007/978-3-030-26948-7_9}}` and save all that typing.  Same for the above.

Reply:
Awesome! Thanks for that tip.

Reply:
That link is behind a springer paywall - the former wasn't.

Reply:
True, yet the DOI is stable. I'm sure folks won't have trouble finding the ePrint version.